src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.xml

```diff
@@ -16,6 +16,7 @@
         <parameter key="router.request_context.host">localhost</parameter>
         <parameter key="router.request_context.scheme">http</parameter>
         <parameter key="router.request_context.base_url"></parameter>
+        <parameter key="router.request_context.base_path">%router.request_context.base_url%</parameter>
```

Comment:
this default is weird, as the base path is a subpart of the base url (the base url may have ``/app_dev.php`` in it)